compatibility/README.md

```diff
@@ -0,0 +1,49 @@
+# Bincode compatibility test
+
+Hello! We are working on releasing bincode 2, and we want to make sure that bincode 2 produces the exact same output as bincode 1.3.
+
+For this we need your help. Please read on if you're using bincode in one of your projects.
+
+We have created a test project that can be used to test compatibility between bincode 1 and 2. It encodes structs with a wide range of settings, checks if the outputs are the same, and then deserializes the struct and checks if the output is the same.
+
+## Adding a test case for your project
+
+To add a test case for your project, please follow the following steps:
+- [ ] Fork https://github.com/bincode-org/bincode
+- [ ] create a new file `compatibility/src/<name>.rs`.
+- [ ] Add a link to your project
+- [ ] Add `Licence: MIT OR Apache-2.0` if you agree to distribute your code under this license
```

Comment:
It feels unclear to me whether you require that test cases be under `MIT OR Apache 2.0` or whether people can license under other licenses too.

Reply:
It's a politically correct way of saying "please license it under `MIT OR Apache-2.0` or else we can't accept your PR"